env.sh.example

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
```

Comment:
How an when we should use this `env.sh` script?  Maybe, it's better to add .env file and use it in docker-compose?

Reply:
Yes env.sh allows someone. Easily create a bash script to export the environment variables.